.syft.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright The Enterprise Contract Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+---
+exclude:
+  - ./acceptance/**
+  - ./internal/documentation/**
+  - ./internal/mocks/**
+  - ./tools/*
```

Comment:
Need to exclude `/tools/kubectl`?

Reply:
That should be in SBOM, its in the image, right?

Reply:
I meant that we should exclude that from the exclude list. IOW, we should make sure `/tools/kubectl` is included in the SBOM. :rofl:

Reply:
Yeah, `/usr/local/bin/kubectl` is in there (I guess from the image), but not `tools/kubectl/go.mod`